.github/workflows/storybook-pages.yml

```diff
@@ -0,0 +1,38 @@
+name: Storybook GitHub Pages
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - trunk
```

Comment:
If we wanted to, we can also limit this to directories and only publish when changes are made in the respective plugins/packages that have GH pages.
That is if we think running on every push to `trunk` is to much. Of-course we can also manually trigger it for the time being or on a weekly basis.